README.md

```diff
@@ -111,38 +73,344 @@ requirements that this script has that may not already be in place:
 If you are running Debian or Ubuntu, you can make sure that these are
 installed by running:
 
-    sudo apt-get install default-jre perl python wget rdiff-backup socat iptables
+`sudo apt-get install default-jre perl python wget rdiff-backup socat iptables`
```

Comment:
Look at the Mastering Markdown page under Examples and the Code tab
https://guides.github.com/features/mastering-markdown/